### ISSUE TITLE
DWAA support for Maya

### ISSUE THREAD

**Issue by tokejepsen** [pypeclub/pype#382]
**Goal**

Render with DWAA EXR compression from Maya.

**Implementation**

Currently the hurdle is making reviews data from DWAA exrs, cause FFMPEG does not have support for this.

**Comment by tokejepsen** [comment]
A temporary workaround would be to have the option of disabling review and thumbnail creation.

@pypeclub/coreteam what was the plan with DWAA support?
Think there was talk about including an OIIO compiled to convert exrs.

**Comment by mkolar** [comment]
Yes the plan was to use OIIO to keep it multiplatform, but we didn't get around to it yet. Probably the simplest way to do it now would be disabling the review as you said.
